### ISSUE TITLE
#1 Card Sharing

### ISSUE THREAD

**Issue by natashaj7** [translation-cards/unhcr-deck#1]
Narrative
- As a Support Worker
- I want to share my translation decks
- So that I can easily give other support workers access to translated material
 
Context
- Card sharing is needed to allow support workers to share a translation so that it only needs to be created and recorded once and can be shared across many devices quickly.
 
Notes
- Card sharing will be done by exporting a zip file to a user's personal email, through whatsapp, or any other supported medium on the Android share menu. This zip file can then be downloaded into another worker's app via the received email.
- Need to figure out how this sharing can effectively be done while taking up as little phone memory as possible, consider local file cleanup of the exported files

In Scope
- Add the sharing icon to app
- Allow all cards to be exported at once
- Support sharing via email, whatsapp, and other apps on android share menu

Out of Scope
- Making sure the cards shared are exported in the correct order
- Sharing specific decks of cards

Acceptance Criteria

1. Add share button
GIVEN I am on the card menu
WHEN I look the an individual card
THEN I see an icon that gives me the option to share a translation card

2. Sharing via android share menu
GIVEN I clicked the share icon
WHEN I see the android share menu
THEN I choose which medium to share the file under
AND the file gets exported to a zip file through that medium

3. Delete .zip file locally
GIVEN I shared a card deck
WHEN the file gets locally stored
THEN it will get automatically deleted once the deck share is complete

**Comment by PatrickDale** [comment]
This issue was moved to translation-cards/translation-cards#1
